config/express.js

```diff
@@ -125,6 +125,31 @@ app.use((req, res, next) => {
     next();
 });
 
+// set security headers
+app.use((req, res, next) => {
+    res.append('X-Frame-Options', 'DENY');
+    res.append('X-Content-Type-Options', 'nosniff');
+    const hsts = req.app.get('hsts');
+    if (hsts && hsts.seconds !== 0) {
+        const directives = [hsts.seconds];
+        if (hsts.preload) {
+            directives.push('preload');
+        }
+        if (hsts['include subdomains']) {
+            directives.push('includeSubDomains');
+        }
+        res.append('Strict-Transport-Security', directives.join('; '));
+    }
+    const defaultCSP = "default-src 'self'; script-src 'self' 'unsafe-inline'; style-src 'self' 'unsafe-inline'; img-src 'self' data:"
```

Comment:
I don't know offhand all of the possible outputs of the various Enketo libraries, but I think we might want to relax this to allow `data:` URLs generally:

```suggestion
    const defaultCSP = "default-src 'self'; script-src 'self' 'unsafe-inline' data:; style-src 'self' 'unsafe-inline' data:; img-src 'self' data:"
```

Reply:
Done